src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -555,9 +555,9 @@ public RestoreCommand(FrameworkName fallbackFramework)
         {
             effectiveRuntimeSpecs.RemoveAll(spec => spec.Name == runtimeName);
 
-            IEnumerable<string> imports = null;
             foreach (var runtimeFile in runtimeFiles)
             {
+                IEnumerable<string> imports = null;
```

Comment:
It looks like the subsequent `if (imports != null)` check will always be false.  Are you sure that the fix isn't instead just to close the `foreach` statement before the following `if (imports != null)` check?